src/control/SConscript

```diff
@@ -79,15 +79,22 @@ def scons():
 
     # if SPDK_PREFIX differs from PREFIX, copy dir so files can be accessed at
     # runtime
-    if denv.subst("$SPDK_PREFIX") != denv.subst("$PREFIX"):
-        for subdir in ["include", "scripts", "fio_plugin"]:
-            sharedir = join("share", "spdk", subdir)
-            tpath = join(denv.subst("$PREFIX"), sharedir)
-            if not check_dir_exists(tpath):
-                denv.Command(
-                    tpath,
-                    join(denv.subst("$SPDK_PREFIX"), sharedir),
-                    Copy("$TARGET", "$SOURCE"))
+    if denv.subst("$SPDK_PREFIX") != "/usr":
+        if denv.subst("$SPDK_PREFIX") != denv.subst("$PREFIX"):
+            for subdir in ["scripts", "fio_plugin"]:
+                sharedir = join("share", "spdk", subdir)
+                tpath = join(denv.subst("$PREFIX"), sharedir)
+                if not check_dir_exists(tpath):
+                    denv.Command(
+                        tpath,
+                        join(denv.subst("$SPDK_PREFIX"), sharedir),
+                        Copy("$TARGET", "$SOURCE"))
+        tpath = join(denv.subst("$PREFIX"), "include")
```

Comment:
have the changes here been verified? looks like behaviour has changed and I don't understand why. if we have changed the relative location of include spdk files the spdk/scripts/setup.sh won't be able to find them. perhaps this has already been tested by @mjean308 ?

Reply:
It's mostly because before the RPM building, we were installing ".../share/" components into 
a common $PREFIX/share but they really need to be installed into $PREFIX/share/$project.

@mjean308 has been verifying these changes.